src/components/Sidenav.vue

```diff
@@ -82,15 +78,15 @@ export default {
   color: floralwhite;
 }
 .sidenav {
-  height: 100vh;
+  height: 100rem;
```

Comment:
The sidenav is now multiple times larger are you sure this is what you wanted?
![image](https://user-images.githubusercontent.com/58952375/111899590-e9774000-8a2d-11eb-8ebb-fbe8e9ec7897.png)

Reply:
i was working on responsivenesss and I'll continue this after that merge. so i suggest that it can stay, since both versions aren't perfect.